string[] targetBody
string[] actionType
string[] object
string[] targetPose

---
int32 result
